test_environments/message_and_event_brokers/kafka/sasl_scram/no_tls/scram_sha_256/README.md

```diff
@@ -0,0 +1,21 @@
+This is a simple setup of Kafka with SASL_SCRAM authentication (SHA-256 algorithm) and without TLS encryption. 
```

Comment:
```suggestion
# Testing Kafka broker with SASL_SCRAM (SHA-256 algorithm) without TLS

This is a simple setup of Kafka with SASL_SCRAM authentication (SHA-256 algorithm) and without TLS encryption. 
```

Reply:
I added `# Setup Kafka broker with SASL SCRAM authentication (SHA-256 algorithm) and without TLS encryption
`